views/apply/dashboard.pug

```diff
@@ -4,20 +4,20 @@ block mainContent
   section(grid='rows')
     h1(rows='1.0')= title
     div(rows='1.5')
-      p= statusMessage.title
-      p= statusMessage.subline
-      if applicationSlug
+      p= statusMessages['status-messages'][statuses.overallStatus].title
```

Comment:
Could we think of another name to avoid `statusMessages['status-messages']`?  Maybe something like `dashboardMessages`?